<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sandeep406701" ignored="0" name="Regression-Suite" tests="87" failures="68" timestamp="09 Jan 2019 06:48:08 GMT" time="42.186" errors="0">
  <testcase name="test_01_verify_placing_an_order" time="2.796" classname="com.rest.api.PostApi"/>
  <testcase name="test_02_verify_placing_an_order_at_specific_time" time="3.745" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.033" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.039" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.034" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.023" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.027" classname="com.rest.api.PostApi"/>
  <testcase name="test_03_verify_order_not_placed_with_invalid_time_format" time="0.023" classname="com.rest.api.PostApi"/>
  <testcase name="test_04_verify_order_not_placed_with_incorrect_location_of_one_stop" time="0.474" classname="com.rest.api.PostApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;400&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PostApi.test_04_verify_order_not_placed_with_incorrect_location_of_one_stop(PostApi.java:122)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_04_verify_order_not_placed_with_incorrect_location_of_one_stop -->
  <testcase name="test_04_verify_order_not_placed_with_incorrect_location_of_one_stop" time="0.448" classname="com.rest.api.PostApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;400&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PostApi.test_04_verify_order_not_placed_with_incorrect_location_of_one_stop(PostApi.java:122)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_04_verify_order_not_placed_with_incorrect_location_of_one_stop -->
  <testcase name="test_05_verify_order_not_placed_with_incorrect_locations_of_stops" time="0.461" classname="com.rest.api.PostApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;400&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PostApi.test_05_verify_order_not_placed_with_incorrect_locations_of_stops(PostApi.java:144)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_placed_with_incorrect_locations_of_stops -->
  <testcase name="test_05_verify_order_not_placed_with_incorrect_locations_of_stops" time="0.342" classname="com.rest.api.PostApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;400&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PostApi.test_05_verify_order_not_placed_with_incorrect_locations_of_stops(PostApi.java:144)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_placed_with_incorrect_locations_of_stops -->
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.034" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.02" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.02" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.032" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.024" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.02" classname="com.rest.api.PostApi"/>
  <testcase name="test_06_verify_order_not_placed_with_incorrect_end_point" time="0.027" classname="com.rest.api.PostApi"/>
  <testcase name="test_07_verify_order_not_placed_in_past_time" time="0.027" classname="com.rest.api.PostApi"/>
  <testcase name="test_08_verify_order_not_placed_with_identical_two_stops" time="0.383" classname="com.rest.api.PostApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;400&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PostApi.test_08_verify_order_not_placed_with_identical_two_stops(PostApi.java:197)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_08_verify_order_not_placed_with_identical_two_stops -->
  <testcase name="test_01_verify_retrieve_valid_order_with_id" time="0.457" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_01_verify_retrieve_valid_order_with_id(GetApi.java:60)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_01_verify_retrieve_valid_order_with_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.342" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.462" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.568" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.358" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.336" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_retrieved_with_invalid_order_id" time="0.348" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_02_verify_order_not_retrieved_with_invalid_order_id(GetApi.java:92)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_retrieved_with_invalid_order_id -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.347" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.343" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.46" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.459" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.347" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.453" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_retrieved_with_invalid_end_point" time="0.461" classname="com.rest.api.GetApi">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.GetApi.test_03_verify_order_not_retrieved_with_invalid_end_point(GetApi.java:117)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_retrieved_with_invalid_end_point -->
  <testcase name="test_01_verify_cancel_an_assigned_order_with_specific_order_id" time="0.354" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_01_verify_cancel_an_assigned_order_with_specific_order_id(PutApiCancelOrder.java:59)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_01_verify_cancel_an_assigned_order_with_specific_order_id -->
  <testcase name="test_02_verify_cancel_an_ongoing_order_with_specific_order_id" time="0.463" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_02_verify_cancel_an_ongoing_order_with_specific_order_id(PutApiCancelOrder.java:102)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_cancel_an_ongoing_order_with_specific_order_id -->
  <testcase name="test_03_verify_order_not_cancelled_for_an_already_completed_order_with_specific_order_id" time="0.337" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_03_verify_order_not_cancelled_for_an_already_completed_order_with_specific_order_id(PutApiCancelOrder.java:152)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_cancelled_for_an_already_completed_order_with_specific_order_id -->
  <testcase name="test_04_verify_order_not_cancelled_an_already_cancelled_order_with_specific_order_id" time="0.472" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_04_verify_order_not_cancelled_an_already_cancelled_order_with_specific_order_id(PutApiCancelOrder.java:203)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_04_verify_order_not_cancelled_an_already_cancelled_order_with_specific_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="0.566" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="0.459" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="0.384" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="0.346" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="3.386" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_05_verify_order_not_cancelled_with_invalid_order_id" time="0.458" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_05_verify_order_not_cancelled_with_invalid_order_id(PutApiCancelOrder.java:249)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_cancelled_with_invalid_order_id -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.462" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.345" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.352" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.343" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.345" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.344" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.36" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_06_verify_order_not_cancelled_with_invalid_end_point" time="0.346" classname="com.rest.api.PutApiCancelOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCancelOrder.test_06_verify_order_not_cancelled_with_invalid_end_point(PutApiCancelOrder.java:271)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_06_verify_order_not_cancelled_with_invalid_end_point -->
  <testcase name="test_07_verify_order_not_cancelled_without_placing_it" time="0.042" classname="com.rest.api.PutApiCancelOrder"/>
  <testcase name="test_01_verify_complete_an_ongoing_order_with_specific_order_id" time="0.754" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_01_verify_complete_an_ongoing_order_with_specific_order_id(PutApiCompleteOrder.java:59)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_01_verify_complete_an_ongoing_order_with_specific_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="0.343" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="0.471" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="3.377" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="0.352" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="0.342" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_completed_with_invalid_order_id" time="0.352" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_02_verify_order_not_completed_with_invalid_order_id(PutApiCompleteOrder.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_completed_with_invalid_order_id -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.345" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.348" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.351" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.351" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="1.244" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.458" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.349" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_completed_with_invalid_end_point" time="0.457" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_03_verify_order_not_completed_with_invalid_end_point(PutApiCompleteOrder.java:138)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_completed_with_invalid_end_point -->
  <testcase name="test_04_verify_order_not_completed_without_placing_it" time="0.022" classname="com.rest.api.PutApiCompleteOrder"/>
  <testcase name="test_05_verify_order_not_completed_for_an_already_completed_order_with_specific_order_id" time="0.324" classname="com.rest.api.PutApiCompleteOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiCompleteOrder.test_05_verify_order_not_completed_for_an_already_completed_order_with_specific_order_id(PutApiCompleteOrder.java:187)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_05_verify_order_not_completed_for_an_already_completed_order_with_specific_order_id -->
  <testcase name="test_01_verify_take_order_with_specific_order_id" time="0.465" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_01_verify_take_order_with_specific_order_id(PutApiTakeOrder.java:60)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_01_verify_take_order_with_specific_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.534" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.405" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.355" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.343" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.462" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_02_verify_order_not_taken_with_invalid_order_id" time="0.458" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_02_verify_order_not_taken_with_invalid_order_id(PutApiTakeOrder.java:94)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_02_verify_order_not_taken_with_invalid_order_id -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.352" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.451" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.799" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.341" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.353" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.346" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.351" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_03_verify_order_not_taken_with_invalid_end_point" time="0.341" classname="com.rest.api.PutApiTakeOrder">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;201&amp;gt;
     but: was &amp;lt;503&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <201>
     but: was <503>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.rest.api.PutApiTakeOrder.test_03_verify_order_not_taken_with_invalid_end_point(PutApiTakeOrder.java:119)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_03_verify_order_not_taken_with_invalid_end_point -->
  <testcase name="test_04_verify_order_not_taken_without_placing_it" time="0.026" classname="com.rest.api.PutApiTakeOrder"/>
</testsuite> <!-- Regression-Suite -->
